GVRf/Framework/src/org/gearvrf/GVRActivity.java

```diff
@@ -136,6 +137,32 @@ public void setScript(GVRScript gvrScript, String distortionDataFileName) {
         }
     }
 
+    /**
+     * Sets whether to force rendering to be single-eye, monoscopic view.
+     * 
+     * @param force
+     *            If true, will create a GVRMonoscopicViewManager when
+     *            {@linkplain setScript setScript()} is called. If false, will
+     *            proceed to auto-detect whether the device supports VR
+     *            rendering and choose the appropriate ViewManager. This call
+     *            will only an effect if it is called before
```

Comment:
`will only an effect` s/b `will only have an effect`, but I think most people can deal with this sort of typo.